keyboards/lily58/keymaps/yshrsmz/keymap.c

```diff
@@ -71,15 +71,15 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  *                   `----------------------------'           '------''--------------------'
  */
 [_LOWER] = LAYOUT(
-  KC_TILD, _______, _______, _______, _______, _______,                    _______, _______, _______,_______, _______, _______,
-  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                      KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12,
+  KC_TILD, S(KC_1), S(KC_2), S(KC_3), S(KC_4), S(KC_5),                   S(KC_6), S(KC_7), S(KC_8),S(KC_9), S(KC_0), S(KC_EQL),
```

Comment:
Just a heads up, we have a bunch of these pre shifted. 

https://docs.qmk.fm/#/keycodes_us_ansi_shifted?id=keycodes

You could use these instead, if you wanted.

Reply:
Thanks, updated.